mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
Above you mentioned documentation. Where and how should this be documented? 

I would also love to have a tool that would verify this behavior. Signalk/testclient could easilyl be extended to convert all deltas to full and run verify against that.